build.sh

```diff
@@ -0,0 +1,4 @@
+# https://stackoverflow.com/questions/59144120/gopath-go-mod-exists-but-should-not-in-aws-elastic-beanstalk
+sudo rm /var/app/current/go.*
```

Comment:
This is where we remove the source?

Reply:
Yes, this is what I was describing on the call last night. I'll make a follow-up change in the near future to switch to just packing and deploying the precompiled binary.